### DISCUSSION TITLE
Unable to submit application

### DISCUSSION THREAD

**Discussion by leesch27** [community/community#125057] [GitHub Education]
### Hi everyone,

I am attempting to apply for GitHub Education for students. Every time I try to submit additional proof, the webpage seems to reload, no matter what proof I attach. I do not get a notification that my application was rejected, nor do I get any errors regarding the attachment. The page just reloads, and the proof I attached gets removed. My student ID does not have a "valid until" date, so I have been using my unofficial and official transcripts as proof. I am also between degrees (just graduated from undergraduate and am going into a master's in the fall), but I am staying at the same institution and am thus keeping my email and ID. Any help would be appreciated.

**Comment by nikhilsbuilds** [comment]
Hello, @leesch27 
Here are a few things you can try:

You can use your Fee Receipt along with the ID card where all the details are clearly visible. I too faced the same problem, there was no  "valid until" date. 

Here is something more you can do.

**Clear cache and cookies**: Sometimes clearing the browser cache and cookies can resolve issues with web pages not functioning correctly. Try clearing them and then attempting to upload the proof again.

**Check file types**: Ensure that the file types you're trying to upload are supported by GitHub Education. Commonly accepted formats are JPG and PNG.

**Resize/compress files**: If the files you're uploading are large, try resizing or compressing them, as there might be a file size limit for uploads.

**Contact GitHub Support**: If none of the above solutions work, your best bet is to contact GitHub Support directly. They should be able to assist you with troubleshooting the issue or provide alternative methods for submitting your proof of enrollment.

**Reply by leesch27** [reply]
1. Cleared cache and cookies, no effect
2. All proof has been either a JPEG or PNG, so issues arent from that
3. Same with file sizes (all files have been below the 1 MB limit)
4. I sent in a request, but I just got an automatic reply sending me the link to apply for benefits and no other information

**Comment by nikhilsbuilds** [comment]
Can you send any screenshot (Don't forget to hide your private details)?

**Reply by leesch27** [reply]
<img width="1377" alt="Screenshot 2024-05-21 at 1 17 16 PM" src="https://github.com/community/community/assets/156934438/90daee80-6ed6-4a06-83c5-9190454cfba9">
<img width="1099" alt="Screenshot 2024-05-21 at 1 17 05 PM" src="https://github.com/community/community/assets/156934438/8f2ddce3-e28b-4ddc-8a99-ed5ccc4cd86d">
It just returns to this screen, removing the file uploaded as proof. No error messages, no acknowledgment that I pushed the "Process my application" button.

**Comment by nikhilsbuilds** [comment]
![image](https://github.com/community/community/assets/100666084/b8b47807-edf0-4af8-9558-03db908e43e5)

check here on your account. is there anything visible ??

**Reply by leesch27** [reply]
Nothing is visible (I'm assuming you are referring to the "You have already submitted 1 request". I do not have that)

**Reply by brittyscience** [reply]
I have the exact SAME problem. I tried to navigate to find a way to contact github support. Unfortunately, due to the categories I selected, it directed me to the git hub education page, or tells me to ask the community. No other options. The knowledge base does not have any solutions to this. I also tried to connect to the "virtual agent" and it also would not connect. I also did all of the troubleshooting the OP did. 

Seeing that this post is two weeks old. Did this issue get resolved for you?

**Reply by leesch27** [reply]
@brittyscience nope, still trying to get access.

**Reply by brittyscience** [reply]
Thank you for getting back to me. I hope we get access soon.

**Reply by Hi-no-to-ri** [reply]
I have the same problem too.
